int32 button_toggle
---
int32 result
